src/test/regress/expected/multi_extension.out

```diff
@@ -24,6 +25,10 @@ BEGIN
         ELSE
             PERFORM pg_sleep(0.1);
         END IF ;
+        -- fail if we reach the end of this loop
+        IF i = 200 THEN
+            raise 'Waited too long for maintenance daemon to start';
```

Comment:
I it's a bit cleaner sense to place this after a loop. That way we can change the 200 only in one place if we ever want to wait longer than 20 seconds.